From b31cc37f5a349e5a4657c96c3b0c645e0228a4ec Mon Sep 17 00:00:00 2001
Message-Id: <b31cc37f5a349e5a4657c96c3b0c645e0228a4ec.1299265003.git.jdenemar@redhat.com>
From: Eric Blake <eblake@redhat.com>
Date: Tue, 1 Mar 2011 10:24:24 -0700
Subject: [PATCH] qemu: avoid overwriting error message

https://bugzilla.redhat.com/show_bug.cgi?id=678870

* src/qemu/qemu_driver.c: Partial backport of
766de43533b7ae2406804e92e83e3ad67df6495d - picks up JUST the code
changes that snuck in amongst all the code motion.
---
 src/qemu/qemu_driver.c |    6 ------
 1 files changed, 0 insertions(+), 6 deletions(-)

diff --git a/src/qemu/qemu_driver.c b/src/qemu/qemu_driver.c
index 73f00d4..73dec53 100644
--- a/src/qemu/qemu_driver.c
+++ b/src/qemu/qemu_driver.c
@@ -6139,8 +6139,6 @@ qemudDomainSaveImageOpen(struct qemud_driver *driver,
     /* Create a domain from this XML */
     if (!(def = virDomainDefParseString(driver->caps, xml,
                                         VIR_DOMAIN_XML_INACTIVE))) {
-        qemuReportError(VIR_ERR_OPERATION_FAILED,
-                        "%s", _("failed to parse XML"));
         goto error;
     }
 
@@ -8505,8 +8503,6 @@ qemudDomainMigratePrepareTunnel(virConnectPtr dconn,
     /* Parse the domain XML. */
     if (!(def = virDomainDefParseString(driver->caps, dom_xml,
                                         VIR_DOMAIN_XML_INACTIVE))) {
-        qemuReportError(VIR_ERR_OPERATION_FAILED,
-                        "%s", _("failed to parse XML"));
         goto cleanup;
     }
 
@@ -8778,8 +8774,6 @@ qemudDomainMigratePrepare2 (virConnectPtr dconn,
     /* Parse the domain XML. */
     if (!(def = virDomainDefParseString(driver->caps, dom_xml,
                                         VIR_DOMAIN_XML_INACTIVE))) {
-        qemuReportError(VIR_ERR_OPERATION_FAILED,
-                        "%s", _("failed to parse XML"));
         goto cleanup;
     }
 
-- 
1.7.4.1

